api/spec/factories/applicant_data_for_matching.rb

```diff
@@ -0,0 +1,14 @@
+# frozen_string_literal: true
+
+FactoryBot.define do
+  factory :applicant_data_for_matching do
+    association :application, factory: :application
+    association :applicant, factory: :applicant
+
+    program { "Computer Science" }
+    department { "Arts and Science" }
+    previous_uoft_ta_experience { "I have none" }
```

Comment:
Should be a bool

Reply:
@George334456 can you add a migration that changes this field to bool in the db?